AsyncDisplayKit/Layout/ASRelativeLayoutSpec.h

```diff
@@ -15,21 +15,21 @@
 /** How the child is positioned within the spec. */
 typedef NS_OPTIONS(NSUInteger, ASRelativeLayoutSpecPosition) {
     /** The child is positioned at point 0 relatively to the layout axis (ie left / top most) */
-    ASRelativeLayoutSpecPositionStart = 0,
+    ASRelativeLayoutSpecPositionStart = 1 << 0,
     /** The child is centered along the specified axis */
-    ASRelativeLayoutSpecPositionCenter = 1 << 0,
+    ASRelativeLayoutSpecPositionCenter = 1 << 1,
     /** The child is positioned at the maximum point of the layout axis (ie right / bottom most) */
-    ASRelativeLayoutSpecPositionEnd = 1 << 1,
+    ASRelativeLayoutSpecPositionEnd = 1 << 2,
```

Comment:
Wow! Since you can't specify multiple options here, this should actually be an `NS_ENUM`, with `=1`, `=2`, `=3` etc.